eip-template.md

```diff
@@ -1,47 +1,57 @@
 ---
-eip: <to be assigned>
 title: <The EIP title is a few words, not a complete sentence>
 description: <Description is one full (short) sentence>
 author: <a comma separated list of the author's or authors' name + GitHub username (in parenthesis), or name and email (in angle brackets).  Example, FirstName LastName (@GitHubUsername), FirstName LastName <foo@bar.com>, FirstName (@GitHubUsername) and GitHubUsername (@GitHubUsername)>
 discussions-to: <URL>
 status: Draft
 type: <Standards Track, Meta, or Informational>
-category (*only required for Standards Track): <Core, Networking, Interface, or ERC>
+category: <Core, Networking, Interface, or ERC> # Only required for Standards Track. Otherwise, remove this field.
 created: <date created on, in ISO 8601 (yyyy-mm-dd) format>
-requires (*optional): <EIP number(s)>
+requires: <EIP number(s)> # Only required when you reference an EIP in the Specification. Otherwise, remove this field.
 ---
 
-This is the suggested template for new EIPs.
+<!-- READ EIP-1 BEFORE USING THIS TEMPLATE! -->
 
-Note that an EIP number will be assigned by an editor. When opening a pull request to submit your EIP, please use an abbreviated title in the filename, `eip-draft_title_abbrev.md`.
+<!-- This is the suggested template for new EIPs. After you have filled in the requisite fields, please delete these comments. -->
 
-The title should be 44 characters or less. It should not repeat the EIP number in title, irrespective of the category. 
+<!-- Note that an EIP number will be assigned by an editor. When opening a pull request to submit your EIP, please use an abbreviated title in the filename, `eip-draft_title_abbrev.md`. -->
```

Comment:
If we want to enforce the number assignment, I'd suggest the first comment to not contain any change suggestion other than change filename. I've run into many times when I propose `eip-some-feature.md` then feedback were given under this file, but when I change file name from `eip-some-feature.md` to `eip-1234.md`, the suggested changes no longer apply.

Reply:
Good feedback - I personally would be a fan of an "@eth-bot assign" command here to avoid that very issue.

Reply:
Not sure how to get around this though - send ideas here.

Reply:
No good idea yet.